index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Currency converter</title>
+  </head>
+  <body>
+    <h2>Add New Currency Rate</h2>
+    <form id="addRateForm">
+      <label for="fromCurrency">From Currency:</label>
+      <input type="text" id="fromCurrency" required />
+
+      <label for="toCurrency">To Currency:</label>
+      <input type="text" id="toCurrency" required />
+
```

Comment:
In your HTML and JavaScript, you are using different variable names for the same concept. in your first form (adding exchange rates) the input fields use` fromCurrency` and `toCurrency`, but in the second form (currency conversion), the input fields use `fromCountry` and `toCountry`, so the same thing (a currency code) is refered to as "Currency" in one form and "Country" in another. Use consistent naming for both forms. Change `fromCountry` and `toCountry` to `fromCurrency` and `toCurrency` everywhere and update it on your Js to make the code consistent and easier to maintain.